.github/pull_request_template.md

```diff
@@ -10,9 +11,14 @@ Pick one of the following:
     - This includes any API change to checkbox-ng/checkbox-support
     - Changes to PXU grammar/field requirements
     - Breaking changes to dependencies in snaps (fwts upgrade for example)
-If your change is to providers it can only be (Infra, BugFix or New)
 
-Example Title: Fixed bugged behaviour of checkbox load config (Bugfix)
+If your change is to providers it can only be (Infra, BugFix or New).
+
+Signed commits are required.
+  - See CONTRIBUTING.md (https://github.com/canonical/checkbox/blob/main/CONTRIBUTING.md#signed-commits-required) for further instructions.
```

Comment:
missed `git add` for the CONTRIBUTING.md?

Reply:
You have to have some belief on this one, it resolves once merged 😄 I couldn't think of a relative link to offer for this.

Reply:
(The section is introduced in this same branch, with that `signed-commits-required` being a translation that GitHub's rendering does automatically for a `## Signed commits required`)